codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -262,7 +262,7 @@ public int onDataRead(final ChannelHandlerContext ctx, int streamId, ByteBuf dat
                 }
 
                 if (endOfStream) {
-                    lifecycleManager.closeRemoteSide(stream, ctx.newSucceededFuture());
+                    lifecycleManager.closeStreamRemote(stream, ctx.newSucceededFuture());
```

Comment:
I actually think `closeRemoteSide` was more descriptive. I'm assuming that you renamed this to be consistent with `closeConnectionRemote`?